### PR TITLE
Formats, examples

### PR DESCRIPTION
**Pull request by cloveras**
Some tweaks for date-time formats, and examples.

Background: https://vippsas.slack.com/archives/CEQM7TPAN/p1669642473612899

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -2354,8 +2349,7 @@ components:
           type: string
           description: |-
             The date when the charge is due to be processed.
-
-            Must be in the format `yyyy-MM-dd`
+            Must be in the format `YYYY-MM-DD`.
```
Comment (skasti) [inline docs/openapi.yaml:2352]: we do NOT want to use week-based year (YYYY vs yyyy) and day-of-year (DD vs dd) AFAIK

https://dev.to/shane/yyyy-vs-yyyy-the-day-the-java-date-formatter-hurt-my-brain-4527

File: docs/openapi.yaml [hunk 2]
```diff
@@ -2403,10 +2397,10 @@ components:
           description: |-
             The date when the charge is due to be processed.
 
-            If the charge is DIRECT_CAPTURE, the charge is processed and charged on due date.
-            If the charge is RESERVE_CAPTURE, the charge is RESERVED on due date.
+            If the charge is `DIRECT_CAPTURE`, the charge is processed and charged on due date.
+            If the charge is `RESERVE_CAPTURE`, the charge is `RESERVED` on due date.
 
-            Must be in the format `yyyy-MM-dd`
+            Must be in the format `YYYY-MM-DD`
```
Comment (skasti) [inline docs/openapi.yaml:2403]: same here. the format is yyyy-MM-dd, not YYYY-MM-DD

File: docs/openapi.yaml (hunk 1 above)
Comment (cloveras) [reply docs/openapi.yaml:2352]: Internal RFC link: https://github.com/vippsas/vipps-rfc/blob/master/docs/0001-api-guidelines.md#parameters

Comment (skasti) [reply docs/openapi.yaml:2352]: Ah, yes, just specify that it is ISO 8601, then it'll be great :D

Comment (cloveras) [inline docs/openapi.yaml:2352]:
```suggestion
            Must be in the format `YYYY-MM-DD` and ISO 8601.
```

File: docs/openapi.yaml (hunk 2 above)
Comment (cloveras) [inline docs/openapi.yaml:2403]:
```suggestion
            Must be in the format `YYYY-MM-DD` and ISO 8601.
```

**Review by ameliemaucade — APPROVED** [review]

**Pull request merged**